### PR TITLE
[3.0][cdc-common] Change DataChangeEvent from interface to class, and add implementation for DataChangeEvent.

### PR DESCRIPTION
**Pull request by joyCurry30**
Refer to #2665 
Change DataChangeEvent from interface to class, and add implementation for DataChangeEvent.
PTAL. @lvyanquan @GOODBOY008

### CODE REVIEW

**Review by GOODBOY008 — APPROVED** [review]
Thanks for you contritbutions, I left a minor comment.

File: flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java [hunk 1]
```diff
@@ -18,24 +18,162 @@
...
+    }
+
+    public Map<String, String> meta() {
+        return meta;
+    }
+
+    /**
+     * Creates a {@link DataChangeEvent} instance that describes the insert event.
+     *
+     * @param tableId table id.
```
Comment (GOODBOY008) [inline flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java:84]: Remove `@param` will be more clean.

**Review by lvyanquan — APPROVED** [review]
+1

File: flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java (hunk 1 above)
Comment (joyCurry30) [reply flink-cdc-common/src/main/java/com/ververica/cdc/common/event/DataChangeEvent.java]: Yes, I have removed them.

**Pull request merged**